docs/reference/connection/index.md

```diff
@@ -0,0 +1,11 @@
+---
+description: How users are connecting to Bifröst and how to control them.
+---
+
+# Connections
+
+Connections representing the way users connecting to Bifröst. Currently, Bifröst is supporting the [SSH protocol](ssh.md), only.
```

Comment:
Connections represent the way users connect to Bifröst. Currently, Bifröst supports exclusively the [SSH protocol](ssh.md).